src/main/java/Greet.java

```diff
@@ -0,0 +1,140 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Collections;
+
+public class Greet {
+    private static int MAX_TASK_SIZE = 100;
+    private static ArrayList<Task> taskList = new ArrayList<Task>(100);
+    private static int listCounter = 0;
+    private static boolean isAwake =true;
+    public static void main(String[] args) {
+        System.out.println("Salutations! my name is Duke\nHow may I help you today?\n");
+        while (isAwake) {
+            Scanner in = new Scanner(System.in);
```

Comment:
Consider using a more descriptive variable name
```suggestion
            Scanner scanner = new Scanner(System.in);
```